{"uid":"a875f0e955c9a97b","name":"test_update_team_with_empty_name_returns_400","fullName":"test.teams.test_put_team#test_update_team_with_empty_name_returns_400","historyId":"6ae4b307c4743ab6f66268eae49e8c27","time":{"start":1728656798694,"stop":1728656798869,"duration":175},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError: Expected status code 400, but got 200. Response text: {\"id\":\"6709359ea38817608\",\"name\":null,\"deleted\":false,\"positionList\":[\"Qa\",\"Dev\",\"Designer\"],\"createdAt\":\"2024-10-11 14:26:38\",\"modifiedAt\":\"2024-10-11 14:26:38\",\"rolesIds\":[\"6703f64ba26bb589d\",\"6703f804da5feb1b0\"],\"rolesNames\":{\"6703f64ba26bb589d\":\"Administrador\",\"6703f804da5feb1b0\":\"Mortal\"},\"layoutSetId\":\"6703f89e1996ee098\",\"layoutSetName\":\"layaout\",\"workingTimeCalendarId\":\"6703f8ace83cd97ed\",\"workingTimeCalendarName\":\"calendar\"}","statusTrace":"setup_teardown_team = {'createdAt': '2024-10-11 14:26:38', 'deleted': False, 'id': '6709359ea38817608', 'layoutSetId': '6703f89e1996ee098', ...}\n\n    @pytest.mark.updateteam\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_update_team_with_empty_name_returns_400(setup_teardown_team):\n        team1 = setup_teardown_team\n        url = EndpointTeams.get_team_id(team1['id'])\n        data = create_team_data(\n            name=\"\")\n        response = TeamService.update_team(url, data)\n>       AssertionManager.assert_status_code_400(response)\n\ntest/teams/test_put_team.py:46: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n        logger.info(f\"Validating status code 400 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 400, (\n                f\"Expected status code 400, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 400, but got 200. Response text: {\"id\":\"6709359ea38817608\",\"name\":null,\"deleted\":false,\"positionList\":[\"Qa\",\"Dev\",\"Designer\"],\"createdAt\":\"2024-10-11 14:26:38\",\"modifiedAt\":\"2024-10-11 14:26:38\",\"rolesIds\":[\"6703f64ba26bb589d\",\"6703f804da5feb1b0\"],\"rolesNames\":{\"6703f64ba26bb589d\":\"Administrador\",\"6703f804da5feb1b0\":\"Mortal\"},\"layoutSetId\":\"6703f89e1996ee098\",\"layoutSetName\":\"layaout\",\"workingTimeCalendarId\":\"6703f8ace83cd97ed\",\"workingTimeCalendarName\":\"calendar\"}\n\ncore/assertions/status_code_assertion.py:37: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1728656773816,"stop":1728656773833,"duration":17},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_teardown_team","time":{"start":1728656798518,"stop":1728656798694,"duration":176},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError: Expected status code 400, but got 200. Response text: {\"id\":\"6709359ea38817608\",\"name\":null,\"deleted\":false,\"positionList\":[\"Qa\",\"Dev\",\"Designer\"],\"createdAt\":\"2024-10-11 14:26:38\",\"modifiedAt\":\"2024-10-11 14:26:38\",\"rolesIds\":[\"6703f64ba26bb589d\",\"6703f804da5feb1b0\"],\"rolesNames\":{\"6703f64ba26bb589d\":\"Administrador\",\"6703f804da5feb1b0\":\"Mortal\"},\"layoutSetId\":\"6703f89e1996ee098\",\"layoutSetName\":\"layaout\",\"workingTimeCalendarId\":\"6703f8ace83cd97ed\",\"workingTimeCalendarName\":\"calendar\"}","statusTrace":"setup_teardown_team = {'createdAt': '2024-10-11 14:26:38', 'deleted': False, 'id': '6709359ea38817608', 'layoutSetId': '6703f89e1996ee098', ...}\n\n    @pytest.mark.updateteam\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_update_team_with_empty_name_returns_400(setup_teardown_team):\n        team1 = setup_teardown_team\n        url = EndpointTeams.get_team_id(team1['id'])\n        data = create_team_data(\n            name=\"\")\n        response = TeamService.update_team(url, data)\n>       AssertionManager.assert_status_code_400(response)\n\ntest/teams/test_put_team.py:46: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n        logger.info(f\"Validating status code 400 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 400, (\n                f\"Expected status code 400, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 400, but got 200. Response text: {\"id\":\"6709359ea38817608\",\"name\":null,\"deleted\":false,\"positionList\":[\"Qa\",\"Dev\",\"Designer\"],\"createdAt\":\"2024-10-11 14:26:38\",\"modifiedAt\":\"2024-10-11 14:26:38\",\"rolesIds\":[\"6703f64ba26bb589d\",\"6703f804da5feb1b0\"],\"rolesNames\":{\"6703f64ba26bb589d\":\"Administrador\",\"6703f804da5feb1b0\":\"Mortal\"},\"layoutSetId\":\"6703f89e1996ee098\",\"layoutSetName\":\"layaout\",\"workingTimeCalendarId\":\"6703f8ace83cd97ed\",\"workingTimeCalendarName\":\"calendar\"}\n\ncore/assertions/status_code_assertion.py:37: AssertionError","steps":[],"attachments":[{"uid":"370627531664fb18","name":"log","source":"370627531664fb18.txt","type":"text/plain","size":5673},{"uid":"b0a60290e9da0ab7","name":"stderr","source":"b0a60290e9da0ab7.txt","type":"text/plain","size":5916}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true},"afterStages":[{"name":"setup_teardown_team::0","time":{"start":1728656798876,"stop":1728656799037,"duration":161},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"updateteam"},{"name":"parentSuite","value":"test.teams"},{"name":"suite","value":"test_put_team"},{"name":"host","value":"fv-az1117-459"},{"name":"thread","value":"2134-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.teams.test_put_team"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["updateteam","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"a875f0e955c9a97b.json","parameterValues":[]}